libs/ledger-live-common/src/bridge/descriptor.test.ts

```diff
@@ -0,0 +1,218 @@
+import { getCryptoCurrencyById } from "@ledgerhq/cryptoassets/currencies";
+import type { CurrencyConfig } from "@ledgerhq/coin-framework/config";
+import { getDescriptor, getSendDescriptor, sendFeatures } from "./descriptor";
+import * as configModule from "../config/index";
+
+jest.mock("../config/index");
+
+describe("getDescriptor", () => {
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it("should return null for undefined currency", () => {
+    const descriptor = getDescriptor(undefined);
+    expect(descriptor).toBeNull();
+  });
+
+  it.each([
+    [
+      "bitcoin",
+      {
+        send: {
+          inputs: {},
+          fees: {
+            hasPresets: true,
+            hasCustom: true,
+            hasCoinControl: true,
+          },
+        },
+      },
+    ],
+    [
+      "ethereum",
+      {
+        send: {
+          inputs: {
+            recipientSupportsDomain: true,
+          },
+          fees: {
+            hasPresets: true,
+            hasCustom: true,
+          },
+        },
+      },
+    ],
+    [
+      "solana",
+      {
+        send: {
+          inputs: {
+            memo: {
+              type: "text",
+              maxLength: 32,
+            },
+          },
+          fees: {
+            hasPresets: true,
+            hasCustom: false,
+          },
+        },
+      },
+    ],
+  ])("should return descriptor for %s", (currencyId, expected) => {
+    const currency = getCryptoCurrencyById(currencyId);
+    jest.spyOn(configModule, "getCurrencyConfiguration").mockReturnValue({
+      status: {
+        type: "active",
+        features: [{ id: "blockchain_txs", status: "active" }],
+      },
+    });
+
+    const descriptor = getDescriptor(currency);
+    expect(descriptor).toMatchObject(expected);
```

Comment:
wdyt of comparing strictly ? We want the config to not have unexpected attributes
```suggestion
    expect(descriptor).toEqual(expected);
```
same for others